### ISSUE TITLE
Cannot save entry with Super Table that contains a nested Matrix

### ISSUE THREAD

**Issue by thegodshatetexas** [engram-design/SuperTable#328]
### Description
Unable to save entry with Matrix field nested inside of a Super Table field. 

![error](https://user-images.githubusercontent.com/881593/73416146-98603600-42d9-11ea-9e5c-c37090a1ea8f.jpg)

### Steps to reproduce
1. Create a super table field with a nested matrix field that has required fields
2. Attach it to a section
3. Save entry

### Additional info

- Plugin version: 2.4.1
- Craft version: 3.4.1

### Application Info

<table class="data fullwidth fixed-layout" dir="ltr">
        <tbody>
                            <tr>
                    <th class="light">PHP version</th>
                    <td>7.3.8</td>
                </tr>
                            <tr>
                    <th class="light">OS version</th>
                    <td>Darwin 19.2.0</td>
                </tr>
                            <tr>
                    <th class="light">Database driver &amp; version</th>
                    <td>MySQL 5.7.26</td>
                </tr>
                            <tr>
                    <th class="light">Image driver &amp; version</th>
                    <td>Imagick 3.4.3 (ImageMagick 6.9.6-2)</td>
                </tr>
                            <tr>
                    <th class="light">Craft edition &amp; version</th>
                    <td>Craft Pro 3.4.1</td>
                </tr>
                            <tr>
                    <th class="light">Yii version</th>
                    <td>2.0.32</td>
                </tr>
                            <tr>
                    <th class="light">Twig version</th>
                    <td>2.12.3</td>
                </tr>
                            <tr>
                    <th class="light">Guzzle version</th>
                    <td>6.5.1</td>
                </tr>
                            <tr>
                    <th class="light">Imagine version</th>
                    <td>1.2.3-dev</td>
                </tr>
                    </tbody>
    </table>

### Plugins
<table class="data fullwidth fixed-layout" dir="ltr">
            <tbody>
                                    <tr>
                        <th class="light">Copy</th>
                        <td>1.0.0-RC9</td>
                    </tr>
                                    <tr>
                        <th class="light">Linkit</th>
                        <td>1.1.11</td>
                    </tr>
                                    <tr>
                        <th class="light">Maps</th>
                        <td>3.7.7</td>
                    </tr>
                                    <tr>
                        <th class="light">Navigation</th>
                        <td>1.3.0</td>
                    </tr>
                                    <tr>
                        <th class="light">Redactor</th>
                        <td>2.5.0</td>
                    </tr>
                                    <tr>
                        <th class="light">Redirect plugin for Craft 3</th>
                        <td>1.0.23</td>
                    </tr>
                                    <tr>
                        <th class="light">Route Map</th>
                        <td>1.1.6</td>
                    </tr>
                                    <tr>
                        <th class="light">SEOmatic</th>
                        <td>3.2.41</td>
                    </tr>
                                    <tr>
                        <th class="light">Super Table</th>
                        <td>2.4.1</td>
                    </tr>
                                    <tr>
                        <th class="light">Typed link field</th>
                        <td>1.0.19</td>
                    </tr>
                            </tbody>
        </table>

**Comment by engram-design** [comment]
Just fixed this for the next release.

To get the fix early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-3#479fe94d25d00bc0b7921b338973c3dd13390532 as 2.4.1.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by thegodshatetexas** [comment]
After updating I am getting the following error when attempting to open an entry with the same Super Table field.

> PHP Compile Error – yii\base\ErrorException
> Cannot redeclare verbb\supertable\elements\db\SuperTableBlockQuery::allowOwnerDrafts()

![yii_base_ErrorException](https://user-images.githubusercontent.com/881593/73464133-0a6d6500-4344-11ea-954c-348c0032cd8e.jpg)

Edit: I am also not able to create a new entry.

**Comment by engram-design** [comment]
Should be fixed in [2.4.2](https://github.com/verbb/super-table/releases/tag/2.4.2)
